tests/multinode.rs

```diff
@@ -91,7 +91,9 @@ fn test_multi_node_validator_catchup_from_zero() {
     logger::setup();
     const N: usize = 5;
     trace!("test_multi_node_validator_catchup_from_zero");
-    let leader = TestNode::new_localhost();
+    let leader_kp = KeyPair::new();
```

Comment:
If there's no precedent for a `_kp` suffix, let's not start it here. Write out `_keypair` please.

Reply:
actually there is: https://github.com/solana-labs/solana/blob/5d936e5c8a2af0920c728cc5c7d98c093f49c01b/src/tvu.rs#L178

Reply:
Cool, change that one too please.